internal/ui/login/static/i18n/en.yaml

```diff
@@ -80,6 +80,24 @@ MfaInitDone:
   Title: Multifcator Verification done
   Description: Multifactor verification successfully done. The multifactor has to be entered on each login, even in the actual authentification process.
 
+MfaInitU2F:
+  Title: Multifactor Setup U2F / WebAuthN
```

Comment:
Describe U2F and WebAuthN?

Reply:
i'd postpone this to later (improving login flow and usability)